Updated geom fields

kgjenkins opened the pull request:
I've updated the 3 geom-related fields:
- locn_geometry
- dcat_bbox
- dcat_centroid

A couple questions:
- I set `locn_geometry` multiplicity to "0-1" -- is that right?  (In the case of multiple polygons, the WKT "MULTIPOLYGON" syntax could be used.) (UPDATE: that is right)
- What do we want to do about the aardvark-id?  I think it makes sense to put bbox between geometry and centroid.  So do we need to reassign all the aardvark-id numbers from that point to the end? (UPDATE: these have been renumbered)

kgjenkins commented:
You can preview these changes here:
https://kgjenkins.github.io/opengeometadata.github.io/docs/aardvarkSchema

rmseifried commented:
@kgjenkins it'd be nice if this table matched the [1.0-->Aardvark conversion table](https://opengeometadata.org/docs/aboutAardvark/#how-do-i-convert-my-version-10-metadata-into-aardvark). In that one, the header for the first column is "#" and "Bounding Box" is assigned number 19 (and the rest below it are bumped up one).

Also, I think the `locn_geometry` field needs another set of eyes. Related to this issue? https://github.com/geoblacklight/geoblacklight/issues/1039

karenmajewicz commented:
New issue for follow up: https://github.com/geoblacklight/geoblacklight-metadata/issues/30

kgjenkins commented:
> @kgjenkins it'd be nice if this table matched the [1.0-->Aardvark conversion table](https://opengeometadata.org/docs/aboutAardvark/#how-do-i-convert-my-version-10-metadata-into-aardvark). In that one, the header for the first column is "#" and "Bounding Box" is assigned number 19 (and the rest below it are bumped up one).

Okay, I'll update this in the PR.

kgjenkins commented:
I've added examples to the `locn_geometry` page:
>Simple bounding box: “ENVELOPE(-111.1, -104.0, 45.0, 40.9)”
Bermuda Triangle: “POLYGON((-80 25, -65 18, -64 33, -80 25))”
Split bounding box for Alaska: “MULTIPOLYGON (((-179.3 51.1, -129.9 51.1, -129.9 71.5, -179.3 71.5, -179.3 51.1)),((172.3 51.2, 179.9 51.2, 179.9 53.1, 172.3 53.1, 172.3 51.2)))”

karenmajewicz reviewed (approved):
This all looks good. I didn't notice before that we had to manually set the nav_order. Maybe another reason to keep the Aardvark order numbers in the chart.
